src/views/pages/name/scottish-limited-partnership-name.njk

```diff
@@ -0,0 +1,150 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.scottishNamePage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.scottishNamePage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: errors.scottish_partnership_name if errors,
+          id: "scottishPartnership_name",
```

Comment:
Have you tested this and does the max length validation work correctly as I think this field needs to be named `partnership_name`?

Reply:
it is supposed to be props.errors.partnership_name

Reply:
id set to : props.errors.partnership_name.